cvat/apps/annotation/migrations/0001_initial.py

```diff
@@ -0,0 +1,98 @@
+# Generated by Django 2.1.9 on 2019-07-04 07:19
+
+import cvat.apps.engine.models
+from django.conf import settings
+import django.core.files.storage
+from django.db import migrations, models
+import django.db.models.deletion
+
+import os
+
+def create_builtins(apps, schema_editor):
+    AnnoDumperModel = apps.get_model('annotation', 'AnnoDumper')
+    AnnoParserModel = apps.get_model('annotation', 'AnnoParser')
+
+    path_prefix = ""
+    if "development" == os.environ.get("DJANGO_CONFIGURATION", "development"):
+        path_prefix = os.path.join("..", "cvat", "apps", "annotation")
+
+    AnnoDumperModel(
+        name="cvat_annotation",
+        file_extension="xml",
+        handler_file=os.path.join(path_prefix, "builtin", "cvat", "dumper.py"),
+        owner=None,
+    ).save()
+
+    AnnoDumperModel(
+        name="cvat_interpolation",
+        file_extension="xml",
+        handler_file=os.path.join(path_prefix, "builtin", "cvat", "dumper.py"),
+        owner=None,
+    ).save()
+
+    AnnoParserModel(
+        name="cvat",
```

Comment:
cvat -> CVAT XML

Reply:
Probably we need `version` as well. In general I will prefer or something like that:
- name = CVAT
- format = XML
- version = 1.1
- specification = [annotation, interpolation]

Reply:
- display name = CVAT XML